lib/connect-session-sequelize.js

```diff
@@ -23,16 +23,14 @@ util.inherits(SequelizeStoreException, Error);
 
 module.exports = function SequelizeSessionInit(Store) {
   function SequelizeStore(options) {
-    options = options || {};
+    this.options = options = options || {};
```

Comment:
i can change it, but is it really worth it? this would just make the constructor harder to read because of all the `this.` prefixes.